### PR TITLE
Frontend header 1st Draft

### PR DESCRIPTION
**Pull request by VasilyShelkov**
Done the first draft with a logo placeholder

### CODE REVIEW

**Comment by VasilyShelkov** [comment]
@Winwardo ready to merge with master

File: src/client/Header.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+const Header = () => {
+	return(
+		<div className="ui three item menu borderless">
```
Comment (Winwardo) [inline src/client/Header.js]: To match with every other Javascript file we have can this use spaces instead of tabs please

**Comment by VasilyShelkov** [comment]
@Winwardo fixed mixed spaces and indents

**Comment by VasilyShelkov** [comment]
for #43

File: public/css/customFonts.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    text-transform: none;
+    line-height: 1;
+
+    /* Better Font Rendering =========== */
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-thumbs-ok:before {
+    font-family: 'thumbsOk' !important;
```
Comment (Winwardo) [inline public/css/customFonts.css:28]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2177439157/issues/source?bid=3148782&fileBranchId=3163784#l28)

**Pull request merged**